mmcv/ops/csrc/pytorch/info.cpp

```diff
@@ -4,15 +4,22 @@
 #include "pytorch_cpp_helper.hpp"
 
 #ifdef MMCV_WITH_CUDA
-#ifndef HIP_DIFF
+#ifdef MMCV_WITH_HIP
+#include <cuda_runtime_api.h>
```

Comment:
Should we include hip runtime here?

Reply:
Yeah I thought about that when I added this code.  So it's fine because hipify will change this header to the hip runtime header.  Though, since it's just below "#if MMCV_WITH_HIP" having "#include <hip/hip_runtime_api.h> explicitly is less confusing.  I'll make that change now!